astropy/table/jsviewer.py

```diff
@@ -75,7 +75,7 @@ class Conf(_config.ConfigNamespace):
     }});
 }});
 </script>
-""" % dict(
+""" % dict(  # noqa: UP031
```

Comment:
Given the conversation in #14583 I think adding this `noqa` is the best option.

The string here is supposed to contain a small bit of javascript and the current string formatting makes it easier to read it as javascript rather than when it is turned into an f-string or `.format` statement.

Reply:
I agree.